jicofo/src/main/kotlin/org/jitsi/jicofo/xmpp/AvModerationHandler.kt

```diff
@@ -80,6 +80,8 @@ class AvModerationHandler(
 
                     val enabled = incomingJson["enabled"] as Boolean?
 
+                    // The message either updates the "enabled" state (if "enabled" is present), or the whitelists (if
+                    // "enabled" is not present).
```

Comment:
It's not possible for a message to carry both enabled state and a whitelist?

Reply:
I asked the same question when Damian wrote the original code. I changed it to support both at the same time :)